packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -98,6 +98,9 @@ expectError(jest.enableAutomock('moduleName'));
 expectType<typeof jest>(jest.isolateModules(() => {}));
 expectError(jest.isolateModules());
 
+expectType<typeof Promise>(jest.isolateModulesAsync(() => {}));
```

Comment:
```suggestion
expectType<Promise<void>>(jest.isolateModulesAsync(() => {}));
```

Reply:
Done in 5810116f02db4f5b0ca78a0f49be40de0f925a1d